static/scss/close-button.scss

```diff
@@ -0,0 +1,19 @@
+.close-button {
```

Comment:
Could this just be added to `static/scss/button.scss` instead of creating a new file?